snapcraft/tests/test_common.py

```diff
@@ -112,8 +111,13 @@ class ArchTestCase(testtools.TestCase):
 
     def setUp(self):
         super().setUp()
-        common.host_machine = platform.machine()
-        common.target_machine = common.host_machine
+        # tests will override host_machine, protect it first
+        ph = patch('snapcraft.common.host_machine', new=common.host_machine)
```

Comment:
thanks to @elopio :wink: we've called this `ph` `patcher` everywhere, mind changing it to keep consistency and for readability?